lib/command.ts

```diff
@@ -231,60 +231,43 @@ export function command(
     if (command) {
       currentContext.commands.push(command);
       currentContext.fullCommands.push(commandHandler.original);
+    } else {
+      // A null command indicates we are running the default command,
+      // if this is the case, we should show the root usage instructions
+      // rather than the usage instructions for the nested default command:
+      yargs.getUsageInstance().freeze();
     }
     const builder = commandHandler.builder;
+
+    let innerYargs: YargsInstance;
     if (isCommandBuilderCallback(builder)) {
-      // a function can be provided, which builds
-      // up a yargs chain and possibly returns it.
       const builderOutput = builder(yargs.reset(parsed.aliases));
-      const innerYargs = isYargsInstance(builderOutput) ? builderOutput : yargs;
-      if (shouldUpdateUsage(innerYargs)) {
-        innerYargs
-          .getUsageInstance()
-          .usage(
-            usageFromParentCommandsCommandHandler(
-              parentCommands,
-              commandHandler
-            ),
-            commandHandler.description
-          );
-      }
-      innerArgv = innerYargs._parseArgs(
-        null,
-        undefined,
-        true,
-        commandIndex,
-        helpOnly
-      );
-      aliases = (innerYargs.parsed as DetailedArguments).aliases;
-    } else if (isCommandBuilderOptionDefinitions(builder)) {
```

Comment:
I'm confused by this code being deleted, since it serves a purpose -- it may be we're just missing tests?